src/webapp/components/collapsible-section/Section.tsx

```diff
@@ -0,0 +1,89 @@
+import { Icon, IconButton, makeStyles, Paper } from "@material-ui/core";
+import React from "react";
+
+type iconPosition = "left" | "right";
+type arrowStyle = "upDown" | "rightLeft";
+
+export interface SectionProps {
+    isOpen?: boolean;
+    setOpen?: (open: boolean) => void;
+    children: React.ReactNode;
+    title: React.ReactNode;
+    collapsible?: boolean;
+    elevation?: number;
+    iconPos?: iconPosition;
+    classProps?: {
+        section?: string;
+        header?: string;
+        content?: string;
+    };
+    arrowStyle?: arrowStyle;
+}
+
+export const Section = ({
+    children,
+    title,
+    collapsible,
+    isOpen = true,
+    setOpen = () => {},
+    elevation = 1,
+    iconPos = "right",
+    classProps = {},
+    arrowStyle = "upDown",
+}: SectionProps) => {
```

Comment:
That's not wrong at all, but typically we prefer to write `props: SectionProps` here and unpack in the first line of the component, so the signature stays single-line and more readable (subjective).